server/routes/items.js

```diff
@@ -63,7 +74,7 @@ route.post('/', upload.single('image'), ( req, res ) => {
     model : req.body.model,
     dimensions    : req.body.dimensions,
     notes : req.body.notes,
-    image    : req.body.image,
+    image    : req.file.path,
```

Comment:
Is this uploading the actual data of the image, or is it just the path to the location of the file? You can not reference the image that was selected on a client. You need to pass the image data.